wpeverest/user-registration
# Enhance - Show all the available PRO fields

### All Submissions:






### Changes proposed in this Pull Request:
When pro version addons are not present or activated, all the available fields are shown, but they are not usable. They will be usable only after the respective plugins are activated.



### How to test the changes in this Pull Request:

1. Deactivate PRO addons
2. Create a new form
3. In the `Fields` section, you should be able to see the pro fields too
4. Try to drag those fields
4. Now, activate PRO addons. They should be now usable.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Show all the available PRO fields, which are disabled.
